db/migrate/20160117211707_add_username_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsernameToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :username, :string, null: false
```

Comment:
What about existing users wit no username?

Note: I'm good with just dropping the production db at this point, despite the bad-practiceness of it all.